openpype/hosts/maya/api/plugin.py

```diff
@@ -180,18 +171,18 @@ def process_reference(self, context, name, namespace, data):
         """To be implemented by subclass"""
         raise NotImplementedError("Must be implemented by subclass")
 
-
     def update(self, container, representation):
 
         import os
         from maya import cmds
+        from openpype.hosts.maya.api.lib import get_container_members
 
         node = container["objectName"]
 
         path = api.get_representation_path(representation)
 
         # Get reference node from container members
-        members = cmds.sets(node, query=True, nodesOnly=True)
+        members = get_container_members(node)
```

Comment:
It was interesting to see that the ReferenceLoader itself wasn't actually considering the members for the new style loaded assets (those that don't have all nodes containerized but only the reference node)  to be collected correctly.

Which means that any code below would find no other node than the single reference node. As such the Alembic related code implementened by @tokejepsen with [this commit](https://github.com/pypeclub/OpenPype/commit/c161a637433bf8b7b0b211dced4c4a99c919ad47) doesn't behave as intended.

Likely it's just getting the reference node (which doesn't have a namespace).
```python
members[0].split(":")[0]
```
This would filter it to just "" because there's no namespace on that particular node.
And as such this code:
```python
alembic_nodes = cmds.ls(
    "{}:*".format(members[0].split(":")[0]), type="AlembicNode"
)
```

Would actually query:
```python
alembic_nodes = cmds.ls(
    ":*", type="AlembicNode"
)
```

I'm guessing that's not going to do what it was intended to do. It's either applying to ALL Alembic Nodes, none at all or all nodes in the top namespace.

This patches it back up that all actual nodes are passed along again. I just wanted to mention since it's been unnoticed for quite a long time now. Since the original change with [this commit](https://github.com/pypeclub/OpenPype/commit/1a950f5c5a5b1cc02d5631c22a598173c7c04304) dates July 26th, 2021

Reply:
I've been able to reproduce the original issue Toke mentioned. @mkolar time for that testing platform and I'll hoop up some tests for OpenPype for this case.

Also found another bug with exactly this code in this PR. Will patch up both.

Reply:
Patched this up.

Also fixes a bug where otherwise it would error with "set" is not subscriptable (you can't index it directly) since `get_container_members()` returns a set. Let me know if anyone would feel more comfortable if instead we'd make that return a list.

Reply:
```python
from maya import cmds


# Create a cube, deform it so it has animated
# geometry deformation
cube = cmds.polyCube(ch=False)[0]
cluster, cluster_handle = cmds.cluster(cube)

cmds.setKeyframe(cluster_handle + ".translateY", t=0, v=0)
cmds.setKeyframe(cluster_handle + ".translateY", t=10, v=10)

# Publish it as pointcache
create("pointcacheDefault", nodes=[cube])
save()
publish()


# Create a new version, so we can update our Alembic
# cache and test it. Let's make a sphere instead
new()   # clear the current scene
sphere = cmds.polySphere(ch=False)[0]
cluster, cluster_handle = cmds.cluster(sphere)

cmds.setKeyframe(cluster_handle + ".translateY", t=0, v=0)
cmds.setKeyframe(cluster_handle + ".translateY", t=10, v=10)


create("pointcacheDefault", nodes=[sphere])
save()
publish()


# Load pointcache latest version (v002)
container = load("pointcacheDefault")

# Change the Alembic Node settings
overrides = {
    "speed": 0.5,
    "offset": 2,
    "cycleType": 3  # Bounce
}
for node in cmds.ls("AlembicNode"):
    for attr, value in overrides.items():
        cmds.setAttr("{}.{}".format(node, attr), value)
    
# Update to old version one
update(container, 1)

# Make sure overrides are preserved
for node in cmds.ls("AlembicNode"):
    for attr, value in overrides.items():
        assert value == cmds.getAttr("{}.{}".format(node, attr))
```
Dropping some psuedocode here for when we get to implement tests. This is basically showing Toke's use case.